core/src/main/java/io/paradoxical/cassieq/discoverable/auth/SignedUrlParameterNames.java

```diff
@@ -51,8 +51,7 @@ public SignedUrlGeneratorBuilder fromSignatureGenerator(SignedUrlSignatureGenera
                     .queueName(signedUrlSignatureGenerator.getQueueName())
                     .startTime(signedUrlSignatureGenerator.getStartDateTime())
                     .endTime(signedUrlSignatureGenerator.getEndDateTime())
-                    .queueName(signedUrlSignatureGenerator.getQueueName())
-                    .sig(signedUrlSignatureGenerator.computeSignature(hmac))
+                    .sig(signedUrlSignatureGenerator.computeSignature(hmac)) // order matters, put this last to keep it pretty
```

Comment:
order matters or is it just to keep it pretty? the two statements are contradictory